warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__rt_feed_guideline_index_vp.sql

```diff
@@ -8,7 +8,6 @@ int_gtfs_quality__rt_feed_guideline_index_vp AS (
     SELECT *
     FROM int_gtfs_quality__rt_feed_guideline_index
     WHERE feed_type = 'vehicle_positions'
-      AND date < CURRENT_DATE
```

Comment:
Curious why removing this

Reply:
answered on slack: it's because I added it erroneously. rt_feed_guideline_index already has this filter